src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -419,7 +420,12 @@ export class PointerTool extends BaseTool {
         }
       }
       return {
-        undoLabel: "drag selection" + (shouldConnect ? " and connect contours" : ""),
+        undoLabel: translate(
```

Comment:
This looks tricky, esp. with the space added to the argument. Wouldn't two distinct keys be easier?

Reply:
Yes, I am totally with you. I tried to be as close as possible to the origin version. I changed it. done.